tests/inlineeditoruiview.js

```diff
@@ -91,6 +101,29 @@ describe( 'InlineEditorUIView', () => {
 			it( 'sets the default value of the #viewportTopOffset attribute', () => {
 				expect( view.viewportTopOffset ).to.equal( 0 );
 			} );
+
+			// Sometimes this test can fail, due to the fact that it have to wait for async ResizeObserver execution.
```

Comment:
Does it still fail despite the timeout?

Reply:
After changing it the same way we changed balloon toolbar tests, it works fine without any timeout 👍